smart-table-module/js/Table.js

```diff
@@ -60,7 +68,8 @@
 
                 var dataRow, oldValue;
 
-                if ((!angular.isArray(array)) || (selectionMode !== 'multiple' && selectionMode !== 'single')) {
+                if ((!angular.isArray(array)) || (selectionMode !== 'multiple' 
+                    && selectionMode !== 'single' && selectionMode !== 'customMultiple')) {
```

Comment:
is Single matter ? Why does the function isModeMutltiple above does not include it ?